src/DssSpell.sol

```diff
@@ -20,76 +20,98 @@ pragma solidity 0.6.12;
+    // Provides a descriptive tag for bot consumption
+    // This should be modified weekly to provide a summary of the actions
+    // Hash: seth keccak -- "$(wget https://<executive-vote-canonical-post> -q -O - 2>/dev/null)"
+    function description() external view returns (string memory) {
+        return SpellAction(action).description();
+    }
 
-    address constant STARKNET_ESCROW_MOM    = 0xc238E3D63DfD677Fa0FA9985576f0945C581A266;
-    address constant STARKNET_ESCROW        = 0x0437465dfb5B79726e35F08559B0cBea55bb585C;
-    address constant STARKNET_DAI_BRIDGE    = 0x659a00c33263d9254Fed382dE81349426C795BB6;
-    address constant STARKNET_GOV_RELAY     = 0x9eed6763BA8D89574af1478748a7FDF8C5236fE0;
+    function officeHours() external view returns (bool) {
+        return SpellAction(action).officeHours();
+    }
 
-    address constant SH_MULTISIG            = 0xc657aC882Fb2D6CcF521801da39e910F8519508d;
-    address constant SH_WALLET              = 0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550;
+    function nextCastTime() external view returns (uint256 castTime) {
+        return SpellAction(action).nextCastTime(eta);
+    }
 
-    address constant FLIPFLOPFLAP_WALLET    = 0x688d508f3a6B0a377e266405A1583B3316f9A2B3;
-    address constant SCHUPPI_WALLET         = 0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167;
-    address constant FEEDBLACKLOOPS_WALLET  = 0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1;
-    address constant MAKERMAN_WALLET        = 0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0;
-    address constant ACREINVEST_WALLET      = 0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D;
-    address constant MONETSUPPLY_WALLET     = 0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF;
-    address constant JUSTINCASE_WALLET      = 0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473;
-    address constant GFXLABS_WALLET         = 0xa6e8772af29b29B9202a073f8E36f447689BEef6;
-    address constant DOO_WALLET             = 0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0;
-    address constant FLIPSIDECRYPTO_WALLET  = 0x62a43123FE71f9764f26554b3F5017627996816a;
-    address constant PENNBLOCKCHAIN_WALLET  = 0x070341aA5Ed571f0FB2c4a5641409B1A46b4961b;
+    // @param _description  A string description of the spell
+    // @param _expiration   The timestamp this spell will expire. (Ex. now + 30 days)
+    // @param _spellAction  The address of the spell action
+    constructor(uint256 _expiration, address _spellAction) public {
+        pause       = PauseAbstract(log.getAddress("MCD_PAUSE"));
+        expiration  = _expiration;
+        action      = _spellAction;
+
+        sig = abi.encodeWithSignature("execute()");
+        bytes32 _tag;                    // Required for assembly access
+        address _action = _spellAction;  // Required for assembly access
+        assembly { _tag := extcodehash(_action) }
+        tag = _tag;
+    }
 
+    function schedule() public {
+        require(now <= expiration, "This contract has expired");
```

Comment:
probably want to switch all of the `now`'s to `block.timestamp`

Reply:
Yeah, I left them as `now` to have as few changes from `DssExec` as possible.  I'm fine to update them here if you guys have checked that diff, but the ultimate place to fix this is in the DssExec PR you're making.